Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -102,6 +102,14 @@
     ears: ClothingHeadsetService
     jumpsuit: ClothingUniformJumpsuitJacketMonkey
     id: PunPunIDCard
+# Harmony gear?
+- type: startingGear
+  id: MobPoquitoGear
+  equipment:
+    ears: ClothingHeadsetService
+    head: ClothingHeadHatChef
+    jumpsuit: ClothingUniformJumpsuitChef
+    id: PoquitoIDcard
```

Comment:
* This prototype is already declared in `Resources/Prototypes/_Harmony/Roles/Jobs/Fun/misc_startinggear.yml`. Revert this change.